openmetadata-ui/src/main/resources/ui/playwright/utils/lineage.ts

```diff
@@ -0,0 +1,386 @@
+/*
+ *  Copyright 2024 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { expect, Page } from '@playwright/test';
+import { get } from 'lodash';
+import { ContainerClass } from '../support/entity/ContainerClass';
+import { DashboardClass } from '../support/entity/DashboardClass';
+import { EntityClass } from '../support/entity/EntityClass';
+import { MlModelClass } from '../support/entity/MlModelClass';
+import { PipelineClass } from '../support/entity/PipelineClass';
+import { SearchIndexClass } from '../support/entity/SearchIndexClass';
+import { TableClass } from '../support/entity/TableClass';
+import { TopicClass } from '../support/entity/TopicClass';
+import { getApiContext, getEntityTypeSearchIndexMapping } from './common';
+
+export const verifyColumnLayerInactive = async (page: Page) => {
+  await page.click('[data-testid="lineage-layer-btn"]'); // Open Layer popover
+  await page.waitForSelector(
+    '[data-testid="lineage-layer-column-btn"]:not(.active)'
+  );
+  await page.click('[data-testid="lineage-layer-btn"]'); // Close Layer popover
+};
+
+export const activateColumnLayer = async (page: Page) => {
+  await page.click('[data-testid="lineage-layer-btn"]');
+  await page.click('[data-testid="lineage-layer-column-btn"]');
+};
+
+export const performZoomOut = async (page: Page) => {
+  for (let i = 0; i < 5; i++) {
+    const zoomOutBtn = page.locator('.react-flow__controls-zoomout');
+    const enabled = await zoomOutBtn.isEnabled();
+    if (enabled) {
+      zoomOutBtn.dispatchEvent('click');
+    }
+  }
+};
+
+export const deleteEdge = async (
+  page: Page,
+  fromNode: EntityClass,
+  toNode: EntityClass
+) => {
+  const fromNodeFqn = get(fromNode, 'entityResponseData.fullyQualifiedName');
+  const toNodeFqn = get(toNode, 'entityResponseData.fullyQualifiedName');
+
+  page.click(`[data-testid="edge-${fromNodeFqn}-${toNodeFqn}"]`, {
+    force: true,
+  });
+
+  if (
+    ['Table', 'Topic'].indexOf(fromNode.getType()) > -1 &&
+    ['Table', 'Topic'].indexOf(toNode.getType()) > -1
+  ) {
+    await page.locator('[data-testid="add-pipeline"]').dispatchEvent('click');
```

Comment:
```suggestion
    await page.click('[data-testid="add-pipeline"]');
```

Reply:
Kept it explicitly. Refer this
https://playwright.dev/docs/input#programmatic-click